### PR TITLE
Add grafana tokens rules

### PR DESCRIPTION
**Pull request by jmatosgrafana**
### Description:
Add rules for the 3 types of Grafana tokens: api key, service account token and cloud api token

### Checklist:

* [x] Does your PR pass tests?
* [x] Have you written new tests for your changes?
* [x] Have you lint your code locally prior to submission?

### CODE REVIEW

File: cmd/generate/config/rules/grafana.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+)
+
+func GrafanaApiKey() *config.Rule {
+	// define rule
+	r := config.Rule{
+		Description: "Grafana api key (or Grafana cloud api key)",
+		RuleID:      "grafana-api-key",
+		SecretGroup: 1,
+		Regex:       generateUniqueTokenRegex(`eyJrIjoi[A-Za-z0-9]{70,}={0,2}`),
+		Keywords:    []string{"eyJrIjoi"},
```
Comment (zricethezav) [inline cmd/generate/config/rules/grafana.go:15]: @jmatosgrafana anyway we could put an upper bound on the `{70,}` quantifier? Not a huge deal but typically for all the rules in the default gitleaks configuration everything is bounded.

File: cmd/generate/config/rules/grafana.go [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+}
+
+func GrafanaCloudApiToken() *config.Rule {
+	// define rule
+	r := config.Rule{
+		Description: "Grafana cloud api token",
+		RuleID:      "grafana-cloud-api-token",
+		SecretGroup: 1,
+		Regex:       generateUniqueTokenRegex(`glc_[A-Za-z0-9+/]{32,}={0,2}`),
+		Keywords:    []string{"glc_"},
```
Comment (zricethezav) [inline cmd/generate/config/rules/grafana.go:34]: same goes for here, could `{32,}` have an upper bound?

**Comment by zricethezav** [comment]
@jmatosgrafana thanks for opening a PR! I left two comments otherwise this LGTM 👍🏻

File: cmd/generate/config/rules/grafana.go (hunk 1 above)
Comment (jmatosgrafana) [reply cmd/generate/config/rules/grafana.go:15]: @zricethezav Thank you for the quick feedback. Here is the answer we provided to GitHub secret scanning team:
_it's a bit hard to predict the largest size but I'd reasonably expect a GRAFANA_API_KEY token not to exceed ~400 characters_

Would 400 be a reasonable upper bound for you ?

File: cmd/generate/config/rules/grafana.go (hunk 2 above)
Comment (jmatosgrafana) [reply cmd/generate/config/rules/grafana.go:34]: Would [400](https://github.com/zricethezav/gitleaks/pull/959#discussion_r950198461) be a reasonable upper bound for you ?

**Comment by zricethezav** [comment]
Superb PR @jmatosgrafana, thanks for contributing

**Pull request merged**